examples/js/controls/TransformControls.js

```diff
@@ -903,7 +913,7 @@
 
 		function onPointerMove( event ) {
 
-			if ( scope.object === undefined || scope.axis === null || _dragging === false || ( event.button !== undefined && event.button !== 0 ) ) return;
```

Comment:
I just remember @elisee putting the code you just removed, but I don't remember why.